projects/testing-library-dom/rollup.config.js

```diff
@@ -0,0 +1,48 @@
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import alias from '@rollup/plugin-alias';
+import replace from '@rollup/plugin-replace';
+
+export default [
```

Comment:
This gets skinnier and skinnier! Now, knowing https://github.com/testing-library/dom-testing-library/issues/413#issuecomment-579834401 I'd almost say that instead of processing the dependency that we replace it with a warning that says you need to be using this library in an environment that supports `MutationObserver` to make the code to manage here even smaller.